tools/test-helpers/helpers.js

```diff
@@ -0,0 +1,35 @@
+window.D2L = window.D2L || {};
+window.D2L.TestHelpers = window.D2L.TestHelpers || {};
+
```

Comment:
Can we avoid using global variables? Like if we're exporting all the helpers then consumers can just import the ones they need: i.e. `import {focus, whatever} from 'helpers.js';`

Reply:
I _think_ in order for them to be available within the `evaluate` scope, they'd need to be on the `window`.  I could try to add a script module to do the import from the Puppeteer side, and it would have to add them to the `window` still.  But then every test file will need to do this.